### ISSUE TITLE
Building fails in cabal-install 2.2.0.0 due to missing line in cabal file

### ISSUE THREAD

**Issue by pechersky** [kazu-yamamoto/unix-time#41]
As identified in https://github.com/haskell/cabal/issues/5169, trying to build `unix-time` with cabal 2.2.0.0 fails at not finding `config.h`. This can be fixed by putting `include-dirs: cbits` in the unix-time.cabal file.

**Comment by pechersky** [comment]
As a stopgap for those that want to `cabal new-build unix-time` or packages that rely on it, here is @mistuke's workaround:

```
git clone https://github.com/kazu-yamamoto/unix-time
cd unix-time
cabal new-build
cd $yourpackagedir
C_INCLUDE_PATH=/home/user/unix-time/dist-newstyle/build/x86_64-linux/ghc-8.0.2/unix-time-0.3.7/build/cbits cabal new-build yourpackage
```

You'll have to change the paths as appropriate for your use case.

**Comment by kazu-yamamoto** [comment]
Let's close this.

**Comment by skeuchel** [comment]
Sorry for re-opening this old issue. Would you consider to also edit the meta-data on hackage here:
[http://hackage.haskell.org/package/unix-time-0.3.7/unix-time.cabal/edit](http://hackage.haskell.org/package/unix-time-0.3.7/unix-time.cabal/edit)

I'm building package databases using cabal with constraints from stackage and this is biting me.

**Comment by kazu-yamamoto** [comment]
I did but I got `Cannot change build information (just the dependency version constraints)`.

Do you know how to fix it? Or, could you use v0.3.8?

**Comment by skeuchel** [comment]
Interesting, I thought hackage would also allow that. Unfortunately I don't know if it's possible then. The hackage trustees could potentially help, but I don't feel strongly about this. I have a workaround in place now.
